src/main/webapp/script.js

```diff
@@ -10,4 +10,23 @@
 // distributed under the License is distributed on an "AS IS" BASIS,
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
-// limitations under the License
\ No newline at end of file
+// limitations under the License
+function requestTranslation() {
+    const text = document.getElementById('text').value;
```

Comment:
A suggestion Cam or Zu made to me once was putting calls to document.getElementById() outside of any function call (i.e. making them global variables). This way, you only traverse the DOM one time to lookup these elements, as opposed to every time your function runs.

Reply:
I'd want the function to get the text value every time it runs though, cuz that means that someone is making a new request to get their stuff translated no?

Reply:
Ohh! Yeah, then I think you're fine with this.